diskmaker/controllers/lv/reconcile_test.go

```diff
@@ -318,34 +347,51 @@ func getDeiveIDs() []string {
 }
 
 func createTmpDir(t *testing.T, dir, prefix string) string {
-	tmpDir, err := ioutil.TempDir(dir, prefix)
+	tmpDir, err := os.MkdirTemp(dir, prefix)
 	if err != nil {
 		t.Fatalf("error creating temp directory : %v", err)
 	}
 	return tmpDir
 }
 
-func createTmpFile(t *testing.T, dir, pattern string) *os.File {
-	tmpFile, err := ioutil.TempFile(dir, pattern)
+func createTmpFile(t *testing.T, dir, pattern string, size int64) *os.File {
+	tmpFile, err := os.CreateTemp(dir, pattern)
 	if err != nil {
 		t.Fatalf("error creating tmp file: %v", err)
 	}
+	err = tmpFile.Truncate(size)
+	if err != nil {
+		t.Fatalf("error truncating tmp file: %v", err)
+	}
 	return tmpFile
 }
 
 func hasFile(t *testing.T, dir, file string) bool {
-	files, err := ioutil.ReadDir(dir)
+	dentries, err := os.ReadDir(dir)
 	if err != nil {
 		t.Fatalf("error reading directory %s : %v", dir, err)
 	}
-	for _, f := range files {
-		if strings.Contains(f.Name(), file) {
+	for _, d := range dentries {
+		if strings.Contains(d.Name(), file) {
 			return true
 		}
 	}
 	return false
 }
 
+func formatAsExt4(t *testing.T, fname string) {
+	cmd := exec.Command("mkfs.ext4", fname)
+	err := cmd.Run()
+	if err != nil {
+		t.Fatalf("error formatting file: %v", fname)
```

Comment:
This would hide the actual command error if any. It's better to show it with something like:

`t.Fatalf("error formatting file: %v Command error: %v", fname, cmd.Err)`